### DISCUSSION TITLE
Is hostd still in pre-release?

### DISCUSSION THREAD

**Discussion by Xalaxis** [SiaFoundation/hostd#106] [Questions & Support] [answered]
I notice that in the most recent release the "beta" tag and the "pre-release" label have been removed, although the release description still says it is in beta. I also first found it by seeing it advertised as the main client on the sia download website. Is hostd safe for use on the mainnet?

**Comment by n8mgr** [comment, marked as answer]
`hostd` is now in beta and considered safe for use on Mainnet. In many ways, it's more stable than `siad`.

**Reply by Xalaxis** [reply]
Thank you for your answer. I'm loving the interface and simplicity so far 🙂 . Please can you update the "current status" section of the README.
